### ISSUE TITLE
Documentation: add examples of queries made using `XPATH`

### ISSUE THREAD

**Issue by ngirard** [MontFerret/ferret#606]
Unless I missed something, the documentation doesn't explain how to query document metadata (searching "site:montferret.dev metadata" through Google returned nothing, neither did grepping the source code).

As an example, I tried to query the `og:url` metadata.
I tried variations of `//meta[property='og:url']::attr(content)`, with or without the leading `//`, and with or without the `attr(content)`, but was unsuccessful.

Cheers

**Comment by ziflex** [comment]
Try to use [XPATH](https://www.montferret.dev/docs/stdlib/html/#xpath) function for that.

**Comment by suntong** [comment]
Yeah, examples of queries made using `XPATH` would be super, whether in the doc or the cookbook.
